serving/docs/lmi_new/user_guides/tnx_user_guide.md

```diff
@@ -0,0 +1,75 @@
+# Transformers-NeuronX Engine in LMI
+
+## Model Artifacts Structure
```

Comment:
Can we add a link back to the model artifact tnx section somewhere in this section? lgtm after we add that

Reply:
Updated